lib/utils.pm

```diff
@@ -101,6 +102,10 @@ sub unlock_if_encrypted {
     }
 }
 
+sub turn_off_packagekit() {
```

Comment:
1. we have logic which doing this only difference that it doing this from console 
2. in any case using x11_start_program during installation looks like bad idea . in case something will go wrong from product side I am not sure that devs will accept this as real case scenario 

suggest to drop this function , select console and stop packagekit there with utils-> pkcon_quit